dist/Storable/Storable.xs

```diff
@@ -5880,6 +5898,50 @@ static SV *retrieve_integer(pTHX_ stcxt_t *cxt, const char *cname)
     return sv;
 }
 
+/*
+ * retrieve_boolean_true
+ *
+ * Retrieve boolean true copy.
+ */
+static SV *retrieve_boolean_true(pTHX_ stcxt_t *cxt, const char *cname)
+{
+    SV *sv;
+    HV *stash;
+
+    TRACEME(("retrieve_boolean_true (#%d)", (int)cxt->tagnum));
+
+    sv = newSVsv(&PL_sv_yes);
+    stash = cname ? gv_stashpv(cname, GV_ADD) : 0;
+    SEEN_NN(sv, stash, 0);  /* Associate this new scalar with tag "tagnum" */
+
+    TRACEME(("boolean true"));
+    TRACEME(("ok (retrieve_boolean at 0x%" UVxf ")", PTR2UV(sv)));
```

Comment:
This and the _false version should probably log the new function names to match all the other similar logging.

Reply:
Fixed.